content/misc/lab-setup.md

```diff
@@ -0,0 +1,24 @@
+# Setting up the Lab environment
+
+If you have already cloned the repository, make sure it is updated:
+
+```console
+cd operating-systems
+git pull --rebase
+```
+
+The command may fail if you have uncommitted changes.
+If that is the case, commit changes and retry.
```

Comment:
Commit or stash and pop after `git pull`? I generally support commits, but last semester's experience showed me that maybe the simpler option is the best / fastest. @Alex-deVis, what do you say?

Reply:
Right, but only say `git stash` -> `git pull` -> `git stash pop`. Give one comprehensive option, not a bunch.

Reply:
This will confuse people. I wouldn't give options. One way to rule them all.

Reply:
To sort this out add the inline suggestion and mark this as resolved.